service/nodeConnectivityChecker.go

```diff
@@ -98,7 +98,13 @@ func (s *service) apiRouter(ctx context.Context) {
 	router.HandleFunc(arrayStatus, connectivityStatus).Methods("GET")
 	router.HandleFunc(arrayStatus+"/"+"{arrayId}", getArrayConnectivityStatus).Methods("GET")
 	//start http server to serve requests
-	err := http.ListenAndServe(apiPort, router)
+	server := &http.Server{
+		Addr:         apiPort,
+		Handler:      router,
+		ReadTimeout:  timeout,
+		WriteTimeout: timeout,
```

Comment:
From where these timeout values are coming?

Reply:
https://github.com/dell/csi-powerscale/blob/main/service/controllerNodeToArrayConnectivity.go#L12

Reply:
ok, another file from the same package.